static/js/components/CustomInput.jsx

```diff
@@ -0,0 +1,51 @@
+import { TextField } from "@material-ui/core";
+
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./Input.css";
+
+const CustomInput = ({
```

Comment:
This whole component is probably unnecessary, given that we are going to restyle anyway.